packages/model-registry/upstream/frontend/src/app/pages/modelCatalog/components/ModelCatalogCard.tsx

```diff
@@ -49,15 +54,23 @@ const ModelCatalogCard: React.FC<ModelCatalogCardProps> = ({ model, source, trun
               />
             )}
             <FlexItem align={{ default: 'alignRight' }}>
-              {isValidated ? (
-                <Popover bodyContent={MODEL_CATALOG_POPOVER_MESSAGES.VALIDATED}>
-                  <Label color="purple" isClickable>
-                    Validated
-                  </Label>
-                </Popover>
-              ) : (
-                source && <Label>{source.name}</Label>
-              )}
+              <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+                {isValidated && (
+                  <Popover bodyContent={MODEL_CATALOG_POPOVER_MESSAGES.VALIDATED}>
+                    <Label color="purple" isClickable>
+                      Validated
+                    </Label>
+                  </Popover>
+                )}
+                {isRedHat && (
+                  <Popover bodyContent={MODEL_CATALOG_POPOVER_MESSAGES.RED_HAT}>
+                    <Label color="grey" isClickable>
+                      Red Hat
```

Comment:
@YuliaKrimerman the design does say "Red Hat AI", the feedback was just that the `MODEL_CATALOG_POPOVER_MESSAGES.RED_HAT_AI` constant wasn't what you exported.